ross/bearing_seal_element.py

```diff
@@ -5,33 +5,13 @@
 import matplotlib.pyplot as plt
 import matplotlib.patches as mpatches
 from abc import ABC
+from ross.ross.element.py import *
```

Comment:
I don't think this import will work. 
Try:
`from element import Element`
In this case is a good practice to be explicit on the imports and not use `import *`

Reply:
Done!

- [shaft_element.py](https://github.com/ross-rotordynamics/ross/commit/14e8f8b2b701d02012c5228baabd522d6897253f)

- [disk_element.py](https://github.com/ross-rotordynamics/ross/commit/690e8ae9f7034da69b4dec034113f323fce429b9)

- [beraing_seal_element.py](https://github.com/ross-rotordynamics/ross/commit/65c71fba65aecd9772d53627faa5bbe5ea807ef5)